### PR TITLE
First round of changes for building and packaging operator.

### PR DESCRIPTION
**Pull request by abhat**

### CODE REVIEW

File: deploy/operator.yaml [hunk 1]
```diff
@@ -16,11 +16,7 @@ spec:
       containers:
         - name: kubefed-operator
           # Replace this with the built image name
-<<<<<<< HEAD
```
Comment (pmorie) [inline deploy/operator.yaml:19]: Looks like a merge conflict

Comment (pmorie) [reply deploy/operator.yaml:19]: I swear I cannot read today. This is fine.

**Comment by abhat** [comment]
@pmorie / approved vs /approve please.

**Pull request merged**